build_runner/lib/src/build_script_generate/build_script_generate.dart

```diff
@@ -18,11 +18,13 @@ import 'builder_ordering.dart';
 
 const scriptLocation = '$entryPointDir/build.dart';
 
-Future<String> generateBuildScript() => logTimedAsync(
-    new Logger('Entrypoint'), 'Generating build script', _generateBuildScript);
+Future<String> generateBuildScript(String configKey) => logTimedAsync(
+    new Logger('Entrypoint'),
+    'Generating build script',
+    () => _generateBuildScript(configKey));
 
-Future<String> _generateBuildScript() async {
-  final builders = await _findBuilderApplications();
+Future<String> _generateBuildScript(String configKey) async {
```

Comment:
nit: I would make this an optional arg to indicate that it can be null

Reply:
Done